src/cart/src/include/gurt/common.h

```diff
@@ -78,6 +78,20 @@ extern "C" {
 /* Check if bit is set in passed val */
 #define D_BIT_IS_SET(val, bit) (((val) & bit) ? 1 : 0)
 
+#if __has_include(<sys/sdt.h>)
```

Comment:
(style)  spaces required around that '<' (ctx:BxV)
(style)  spaces required around that '>' (ctx:VxB)